minimint-bridge/src/api.rs

```diff
@@ -376,7 +379,7 @@ pub fn decode_invoice(bolt11: String) -> Result<BridgeInvoice> {
     })
 }
 
-pub fn decode_invoice_inner(invoice: &Invoice) -> anyhow::Result<BridgeInvoice> {
+fn decode_invoice_inner(invoice: &Invoice) -> anyhow::Result<BridgeInvoice> {
```

Comment:
this was the problem. should have been private function. when it's pub then flutteR_rust_bridge tries to make bindings.